rasa/nlu/__init__.py

```diff
@@ -2,8 +2,6 @@
 
 import rasa
 from rasa.nlu.train import train
-from rasa.nlu.test import run_evaluation as test
```

Comment:
why did you remove these?

Reply:
If this line is in the code when running `rasa train nlu`, Rasa breaks with this error:
```
ImportError: cannot import name 'get_model' from 'rasa.model' (/home/arthur/Documents/rasa/rasa/rasa/model.py)
```
by removing this line, Rasa works and I didn't find any other issue after this removal